src/eddymotion/model/base.py

```diff
@@ -409,6 +412,119 @@ class DKIModel(BaseModel):
     _model_class = "dipy.reconst.dki.DiffusionKurtosisModel"
 
 
+class GaussianProcessModel:
+    """A Gaussian Process model based on [Andersson16a]_ (fig 1).
+    DWIs need to be transformed to a single ref space (fig 2 [Andersson16b]_ ?)
+
+    Definitions:
+    s: reference/undistorted space: used to denote the space or any image in
+    that space
+    f: observed/distorted image: used to denote any image in acquisition space
+    a: acquisition parameters: PE‐direction and bandwidth in PE‐direction
+    r: rigid body (subject movement) parameters
+    \beta: Eddy current parameters
+    e(\beta): Eddy current‐induced off resonance field (Hz)
+    h: Susceptibility induced off‐resonance field (Hz)
+
+    (fig 1) and algorithm:
+    1. Input: N DWI volumes f_{i} with acq parameters a_{i}; susceptibility
+    field h
```

Comment:
```suggestion
    1. Input: N DWI volumes f_{i}, beta_{i}, r_i{i}, and hyperparameters.
```

Reply:
Removing all this in 5d76b88. I will document properly when we reach a more solid basis. Also, these do not belong to the GP process (Anderson 2015), strictly but to the correction step (Anderson 2016).